Cargo.toml

```diff
@@ -9,7 +9,7 @@ categories = ["multimedia::images"]
 keywords = ["jpg", "jpeg", "encoder", "image"]
 readme = "README.md"
 repository = "https://github.com/vstroebel/jpeg-encoder"
-rust-version = "1.61"
+rust-version = "1.86"
```

Comment:
You need to bump to `1.87` for all the safe arch intrinsics
https://blog.rust-lang.org/2025/05/15/Rust-1.87.0/#safe-architecture-intrinsics
`1.86` was only the target feature change, so the intrinsics still need `unsafe` blocks to compile as shown in this CE link
https://rust.godbolt.org/z/cq15EszcM

https://doc.rust-lang.org/1.86.0/core/arch/x86/fn._mm256_set1_epi32.html
https://doc.rust-lang.org/1.87.0/core/arch/x86/fn._mm256_set1_epi32.html
```rust
// 1.86
pub unsafe fn _mm256_set1_epi32(a: i32) -> __m256i
// 1.87
pub fn _mm256_set1_epi32(a: i32) -> __m256i
```

There don't seem to be any places where `as_chunks` could benefit us with auto-vectorization so I wouldn't recommend bumping to `1.88` in this instance.

Reply:
Good catch, thanks! Fixed.